Helper/Search/Filter.php

```diff
@@ -159,8 +163,10 @@ public function handleRequest(Request $request): void
         }
     }
 
-    public function handleAggregation(array $aggregation)
+    public function handleAggregation(array $aggregation, array $types = [], array $queryFilters = [])
     {
+        $this->queryTypes = $types;
```

Comment:
Filters are created from the search object, and this object knows all types. Why not passing it on construct.
The queryFilter should only be passed.

Reply:
Fixed in https://github.com/ems-project/EMSClientHelperBundle/pull/144